### PR TITLE
chore(alias): add some alias & order by file directory order

### PR DESCRIPTION
**Pull request by sudongyuer**

### CODE REVIEW

File: alias.ts [hunk 1]
```diff
@@ -3,29 +3,35 @@ import { resolve } from 'path'
...
+  '@unocss/shared-integration': r('./packages/shared-integration/src/'),
+  '@unocss/transformer-attributify-jsx': r('./packages/transformer-attributify-jsx/src/'),
+  '@unocss/transformer-compile-class': r('./packages/transformer-compile-class/src/'),
   '@unocss/transformer-directives': r('./packages/transformer-directives/src/'),
   '@unocss/transformer-variant-group': r('./packages/transformer-variant-group/src/'),
-  '@unocss/transformer-compile-class': r('./packages/transformer-compile-class/src/'),
-  '@unocss/transformer-attributify-jsx': r('./packages/transformer-attributify-jsx/src/'),
+  'unocss': r('./packages/unocss/src/'),
   '@unocss/vite': r('./packages/vite/src/'),
+  '@vscode/unocss': r('./packages/vscode/src/'),
```
Comment (antfu) [inline alias.ts:35]: This won't be imported, shouldn't include it

File: alias.ts [hunk 2]
```diff
@@ -3,29 +3,35 @@ import { resolve } from 'path'
...
+  '@unocss/shared-common': r('./packages/shared-common/src/'),
+  '@unocss/shared-docs': r('./packages/shared-docs/src/'),
+  '@unocss/shared-integration': r('./packages/shared-integration/src/'),
+  '@unocss/transformer-attributify-jsx': r('./packages/transformer-attributify-jsx/src/'),
+  '@unocss/transformer-compile-class': r('./packages/transformer-compile-class/src/'),
   '@unocss/transformer-directives': r('./packages/transformer-directives/src/'),
   '@unocss/transformer-variant-group': r('./packages/transformer-variant-group/src/'),
-  '@unocss/transformer-compile-class': r('./packages/transformer-compile-class/src/'),
-  '@unocss/transformer-attributify-jsx': r('./packages/transformer-attributify-jsx/src/'),
+  'unocss': r('./packages/unocss/src/'),
```
Comment (antfu) [inline alias.ts:33]: Why don't we just sort alphabetically?

Comment (sudongyuer) [reply alias.ts:33]: 🐻‍❄️ I think it's easy for contributors to know where the alias should go
image

**Review by antfu — APPROVED** [review]

File: alias.ts (hunk 2 above)
Comment (sudongyuer) [reply alias.ts:33]: > 

Before that, I'd also like to implement a Node script that implements it 😂, If you want to do that, call me. I would like to do it

**Comment by antfu** [comment]
Test failing

**Pull request closed without merging**